src/ui/ResultList/ResultList.ts

```diff
@@ -251,7 +251,14 @@ export class ResultList extends Component {
     layout: ComponentOptions.buildStringOption({
       defaultValue: 'list',
       required: true
-    })
+    }),
+
+    /**
+     * If this option is set to `true`, after changing page, a facet option or the number of results per page, the page will scroll to the top.
```

Comment:
Suggestion:

> Whether to scroll back to the top of the page whenever the end-user interacts with a facet or with the pager.
>
> **Default:** `true`

I now try to start all boolean option descriptions with "Whether". For other types, I either start with "The", or "A/An" in certain cases (e.g., when an option is setting something less "definite", such as a custom function).

There are some style discrepancies in the documentation right now, but hopefully we'll get rid of them all eventually :)